projects/adf-office-services-ext/tsconfig.lib.prod.json

```diff
@@ -1,7 +1,12 @@
 {
   "extends": "./tsconfig.lib.json",
   "compilerOptions": {
-    "declarationMap": false
+    "declarationMap": false,
+    "paths": {
```

Comment:
are you 100% sure this is needed ?

Reply:
it is needed, if we wont add it, typescript will inherit paths from `./tsconfig.lib.json` and for e.g. `@alfresco/aca-shared` we will try to access typescript files from `projects/aca-shared` which results with build error (we cannot build libs with files located outside of the build library root)